### PR TITLE
CKEditor: improved notice when same images is uploaded multiple times…

### PR DESCRIPTION
**Pull request by ashklianko**
… in same HtmlArea #558

### CODE REVIEW

File: src/main/resources/assets/admin/common/js/util/htmlarea/editor/HTMLAreaBuilderCKE.ts [hunk 1]
```diff
@@ -245,14 +245,17 @@ module api.util.htmlarea.editor {
...
-                const xhr = fileLoader.xhr;
 
-                xhr.open('POST', fileLoader.uploadUrl, true);
-                formData.append('file', fileLoader.file, fileLoader.fileName);
-                formData.set('parent', this.content.getPath().toString());
-                formData.set('name', fileLoader.fileName);
-                fileLoader.xhr.send(formData);
+                this.isFileExisting(fileLoader.fileName).then((exists: boolean) => {
+                    if (exists) {
+                        NotifyManager.get().showWarning(`File ${fileLoader.fileName} already exists!`);
```
Comment (alansemenov) [inline src/main/resources/assets/admin/common/js/util/htmlarea/editor/HTMLAreaBuilderCKE.ts]: This warning should be localised/translated

Comment (ashklianko) [reply src/main/resources/assets/admin/common/js/util/htmlarea/editor/HTMLAreaBuilderCKE.ts]: Ok!

File: src/main/resources/assets/admin/common/js/util/htmlarea/editor/HTMLAreaBuilderCKE.ts [hunk 2]
```diff
@@ -281,6 +284,29 @@ module api.util.htmlarea.editor {
             });
         }
 
+        private isFileExisting(fileName: string): wemQ.Promise<boolean> {
```
Comment (alansemenov) [inline src/main/resources/assets/admin/common/js/util/htmlarea/editor/HTMLAreaBuilderCKE.ts]: Rename this to "fileExists"

Comment (ashklianko) [reply src/main/resources/assets/admin/common/js/util/htmlarea/editor/HTMLAreaBuilderCKE.ts]: Updated

File: src/main/resources/assets/admin/common/js/util/htmlarea/editor/HTMLAreaBuilderCKE.ts (hunk 1 above)
Comment (ashklianko) [reply src/main/resources/assets/admin/common/js/util/htmlarea/editor/HTMLAreaBuilderCKE.ts]: https://github.com/enonic/app-contentstudio/pull/193

**Comment by ashklianko** [comment]
Updated

**Pull request merged**